Tests/SqliteUserModelTest.php

```diff
@@ -0,0 +1,106 @@
+<?php
+/**
+ * @copyright  Copyright (C) 2005 - 2017 Open Source Matters, Inc. All rights reserved.
+ * @license    GNU General Public License version 2 or later; see LICENSE
+ */
+
+namespace Joomla\Database\Tests;
+
+use Joomla\Entity\Tests\Helpers\SqliteCase;
+use Joomla\Entity\Tests\Models\User;
+
+/**
+ *
+ * @since  1.0
+ */
+class SqliteUserModelTest extends SqliteCase
+{
+
+	/**
+	 * @return void
+	 */
+	public function testFind()
+	{
+		$model = new User(self::$driver);
+		$id = $model->findLast()->id;
+		$user = $model->find($id);
+
+		$this->assertNotEmpty(
+			$user->getAttributes()
+		);
+
+	}
+
+	/**
+	 * @return void
+	 */
+	public function testFindLast()
+	{
+		$model = new User(self::$driver);
+		$user = $model->findLast();
+
+		$this->assertGreaterThanOrEqual(
+			0,
```

Comment:
Check for the actual value from the dataset

Reply:
I'm uncertain about how to implement some tests. For example for `testIncrement` should I write an sql script and run in on the mockDatabase in order to see that the update has been made?